### PR TITLE
Fix todos

### PR DESCRIPTION
**Pull request by Oshgnacknak**

### CODE REVIEW

File: shared/src/main/scala/webapp/Project.scala [hunk 1]
```diff
@@ -30,9 +31,9 @@ case class Project(
     this.merge(diffSetAccountName)
   }
 
-  def withAddedMaxHours(addedMaxHours: Int) = {
+  def withMaxHours(maxHours: Int) = {
     val diffSetMaxHours =
-      Project.empty.copy(_maxHours = _maxHours.add(addedMaxHours)(using PermIdMutate.withID(myReplicaID)))
+      Project.empty.copy(_maxHours = PosNegCounter.zero.add(maxHours)(using PermIdMutate.withID(myReplicaID)))
```
Comment (mohe2015) [inline shared/src/main/scala/webapp/Project.scala:36]: I'm pretty sure this does not work with a PosNegCounter as it is represented internally as
```json
{
  "myid": {
    "added": 5,
    "removed": 2,
  }
}
```
and setting the `added` value to a lower value will not update. To lower the value you need to increase the `removed` value.

**Review by mohe2015 — CHANGES_REQUESTED** [review]

File: shared/src/main/scala/webapp/Project.scala (hunk 1 above)
Comment (Oshgnacknak) [reply shared/src/main/scala/webapp/Project.scala:36]: We need absolute values according to ui.
We could add the correct difference to arrive at the correct value.

Comment (mohe2015) [reply shared/src/main/scala/webapp/Project.scala:36]: Or just change it to TimedVal for now

Comment (mohe2015) [reply shared/src/main/scala/webapp/Project.scala:36]: Or such a "great" UI as our time tracking tool but I think TimedVal should be good enough for now

Comment (Oshgnacknak) [reply shared/src/main/scala/webapp/Project.scala]: timed val it is

**Review by mohe2015 — APPROVED** [review]
LGTM code-wise! Nice imports cleanup and reducing duplication.

**Pull request merged**